app/test/socobo-inventory/socobo-inventory-create.html

```diff
@@ -66,7 +67,9 @@
       });
 
       test('create button is hidden when in edit mode', function(done) {
-        dialog.show({});
+        var item = {measurement: {value: 'pcs'}, desc: 'test item', info: 'test info'};
```

Comment:
I know that it was implemented like this originially in this PR, but are we agreeing to save the measurements as objects or would it make more sense to simply store the strings?